docs/examples/15_minutes_to_QCoDeS.ipynb

```diff
@@ -650,7 +678,14 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Before starting a measurement we first initialise a database in the location specified by the config object (read the config section below). The db is created with the latest supported version. If the database already exists then that database is used but is upgraded to the latest version."
+    "Before starting a measurement, we first initialise a database. The location of the database is specified by the configuration object of the QCoDeS installation. The database is created with the latest supported version complying with the QCoDeS version that is currently under use. If a database already exists but an upgrade has been done to the QCoDeS, then that database can continue to be used and it is going to be upgraded to the latest version automatically at first connection."
```

Comment:
@GateBuilder may i kindly ask to advertise `initialise_or_create_database_at` function here as well?

Reply:
Sure. 👍